lib/modules/apostrophe-attachments/lib/api.js

```diff
@@ -432,6 +432,8 @@ module.exports = function(self, options) {
         // that cropping coordinates stored in an apostrophe-images widget
         // are passed through when we make a simple call to
         // apos.attachments.url with the returned object
+        // Additionally, pass the `description`, `credit`, and `creditUrl`
```

Comment:
I meant the documentation comment at the start of the entire method. That is parsed as markdown and rendered in the documentation. Should include what the properties actually get called.